src/rendering.js

```diff
@@ -35,6 +37,11 @@ export default function reducer(state = initialState, action) {
         .set('disableScripts', action.disableScriptsAndStyles);
     case SET_RENDER_PARTIAL_ONLY:
       return state.set('renderPartialOnly', action.renderPartialOnly);
+    case SET_RENDER_TEXT_ONLY:
+      return state.withMutations((newState) => newState
+        .setIn(['renderTextOnly', 'setTextOnly'], action.setTextOnly)
+        .setIn(['renderTextOnly', 'tagReplacement'], action.tagReplacement)
```

Comment:
Does this replace all tags or act as a separator between text nodes?

Reply:
it replaces all tags, so if you add the a space for example. it will add it at the beginning and the end of the text. i.e.

`' https://intranet-origin-dev.example.com/some-api/v1  Hello! '`

if tag replacement is not passed:

`https://intranet-origin-dev.example.com/some-api/v1Hello!`

https://github.com/ericnorris/striptags